collect_app/src/main/java/org/odk/collect/android/utilities/FileUtils.java

```diff
@@ -180,69 +167,62 @@ public static String getMd5Hash(InputStream is) {
         }
     }
 
-
-    public static Bitmap getBitmapScaledToDisplay(File f, int screenHeight, int screenWidth) {
-        // Determine image size of f
-        BitmapFactory.Options o = new BitmapFactory.Options();
-        o.inJustDecodeBounds = true;
-        getBitmap(f.getAbsolutePath(), o);
-
-        int heightScale = o.outHeight / screenHeight;
-        int widthScale = o.outWidth / screenWidth;
-
-        // Powers of 2 work faster, sometimes, according to the doc.
-        // We're just doing closest size that still fills the screen.
-        int scale = Math.max(widthScale, heightScale);
-
-        // get bitmap with scale ( < 1 is the same as 1)
-        BitmapFactory.Options options = new BitmapFactory.Options();
-        options.inInputShareable = true;
-        options.inPurgeable = true;
-        options.inSampleSize = scale;
-        Bitmap b = getBitmap(f.getAbsolutePath(), options);
-        if (b != null) {
-            Timber.i("Screen is %dx%d.  Image has been scaled down by %d to %dx%d",
-                    screenHeight, screenWidth, scale, b.getHeight(), b.getWidth());
-        }
-        return b;
-    }
-
     /**
-     * With this method we do not care about efficient decoding and focus
-     * more on a precise scaling to maximize use of space on the screen
+     * Scales image according to the given display
+     *
+     * @param file            containing the image
+     * @param screenHeight    height of the display
+     * @param screenWidth     width of the display
+     * @param scaleAccurately determines whether to scale more precisely without caring about
+     *                        efficient decoding and to maximize use of space on the screen
+     * @return scaled bitmap
```

Comment:
Very nice doc comment here.

Reply:
Do you think it would be helpful if we rename `scaleAccurately` to `upScalingEnabled` with default value as `false` and a comment explaining that `this parameter decides whether the scale should be scaled up or if the window size is greater than the image`?

Reply:
I’d call it `upscale`, but sure, more precise and detailed is usually better.

Reply:
:+1: